MvvmCross/ViewModels/IMvxAppStart.cs

```diff
@@ -6,7 +6,7 @@ namespace MvvmCross.ViewModels
 {
     public interface IMvxAppStart
     {
-        void Start(object hint = null);
+        object Start(object hint = null);
```

Comment:
Is there a good reason to return the updated hint here?

Reply:
Oops, that's just me going crazy on the copy-paste!  Reverting...